.github/workflows/bump-deps.yaml

```diff
@@ -0,0 +1,64 @@
+name: Bump DEPS
```

Comment:
We'd need an explanation how to use this. It can be one sentence for webui, one for cli, but we need something to point users to. Perhaps an existing README in brave-core? Or a wiki? I think a comment here would not be discoverable enough, but we could add a comment here and link to it from elsewhere, could be a good option.

Reply:
Added in https://github.com/brave/brave-core/pull/16068/commits/278a5a9cb172770f6ebfdc3eb55db607324a40bc
W which we can link to later on

Reply:
We should probably mention it somewhere for discoverability. README/wiki/etc.

Reply:
Comment added

Reply:
> Comment added

Where do you mean?

Reply:
Added https://github.com/brave/brave-browser/wiki/Auto-bump-a-dependency-in-DEPS